### ISSUE TITLE
IE11: afterPaste event won't trigger when an image is pasted

### ISSUE THREAD

**Issue by corrrso** [xdan/jodit#341]
Jodit Version:  3.3.23

Browser:  Internet Explorer 11
OS:  Windows 10

Code:

```
`window.editor = new Jodit('#editor', {
			width: '100%',
			height: '100%',
			showWordsCounter: false,
			showCharsCounter: false,
			askBeforePasteFromWord: false,
			askBeforePasteHTML: false,
			language: 'it',
			enter: 'DIV',
			autofocus: false,
			"toolbarSticky": true,
			//toolbarStickyOffset: 30,
			buttons:['appletZoom', //this is a custom button
					 'source',
					'|',
					'font',
					'fontsize',
					'|',
					'bold',
					'strikethrough',
					'underline',
					'italic',
					'|',
					'superscript',
					'subscript',
					'|',
					'brush',
					'paragraph',
					'|',
					'ul',
					'ol',
					'outdent',
					'indent',
					'|',
					'align',
					'undo',
					'redo',
					'|',
					'image',
					'file',
					'video',
					'table',
					'link',
					'|',
					'cut',
					'hr',
					'eraser',
					'copyformat',
					'|',
					'symbol',
					'selectall',
					'print',
					'about'],
			buttonsSM:['appletZoom',
					'source',
					'|',
					'font',
					'fontsize',
					'|',
					'bold',
					'strikethrough',
					'underline',
					'italic',
					'|',
					'superscript',
					'subscript',
					'|',
					'brush',
					'paragraph',
					'|',
					'ul',
					'ol',
					'outdent',
					'indent',
					'|',
					'align',
					'undo',
					'redo',
					'|',
					'image',
					'file',
					'video',
					'table',
					'link',
					'|',
					'cut',
					'hr',
					'eraser',
					'copyformat',
					'|',
					'symbol',
					'selectall',
					'print',
					'about'],
			buttonsMD:['appletZoom',
					'source',
					'|',
					'font',
					'fontsize',
					'|',
					'bold',
					'strikethrough',
					'underline',
					'italic',
					'|',
					'superscript',
					'subscript',
					'|',
					'brush',
					'paragraph',
					'|',
					'ul',
					'ol',
					'outdent',
					'indent',
					'|',
					'align',
					'undo',
					'redo',
					'|',
					'image',
					'file',
					'video',
					'table',
					'link',
					'|',
					'cut',
					'hr',
					'eraser',
					'copyformat',
					'|',
					'symbol',
					'selectall',
					'print',
					'about'],
			buttonsXS:['appletZoom',
					'source',
					'|',
					'font',
					'fontsize',
					'|',
					'bold',
					'strikethrough',
					'underline',
					'italic',
					'|',
					'superscript',
					'subscript',
					'|',
					'brush',
					'paragraph',
					'|',
					'ul',
					'ol',
					'outdent',
					'indent',
					'|',
					'align',
					'undo',
					'redo',
					'|',
					'image',
					'file',
					'video',
					'table',
					'link',
					'|',
					'cut',
					'hr',
					'eraser',
					'copyformat',
					'|',
					'symbol',
					'selectall',
					'print',
					'about'],
			"disablePlugins": "stat,xpath"//source	
		});

editor.events.on('afterPaste', function (e) { /* stuff (it stops before getting here */})`
```

If i copy-paste a local image (snipping tool for instance) by pressing ctrl+v or by pressing "right click + paste", i get this error inside jodit.min.js:

`e.trim=function(t){return t.replace(i.SPACE_REG_EXP_START,"").replace(i.SPACE_REG_EXP_END,"")}`

> JavaScript run-time error: impossible to retrieve property 'replace' of null or undefined (row 9, column 95956)

If i copy-paste some text, everything is fine.

I am sorry for my continuous issues, but the product is absolutely great and resolving these problems will make it perfect!

**Comment by corrrso** [comment]
HI. I have further informations about the error.

Here is the code that creates an exception inside jodit.js:
![image](https://user-images.githubusercontent.com/23524581/74644814-30ce3580-5177-11ea-81b1-31ccb283676f.png)

the parameter "value" is Null.

Here is the call stack (Riga means "row")

![image](https://user-images.githubusercontent.com/23524581/74644870-48a5b980-5177-11ea-81a1-7c612078cbef.png)

**Comment by corrrso** [comment]
@xdan I found the solution. Looks like it's enough to check if clipboard_html is null or not at line 8369 inside jodit.js. Like this pasting both images and text seems to work. Make sure to double-check it tho.

![image](https://user-images.githubusercontent.com/23524581/74659302-5c134d80-5194-11ea-917c-c0955f73f1b6.png)

**Comment by xdan** [comment]
Fixed in 3.3.23
